### PR TITLE
Update firstuseauthenticator.py

### PR DESCRIPTION
**Pull request by efedorov-dart**
Fix error 500 when press change password button.
Disscussed in https://github.com/jupyterhub/the-littlest-jupyterhub/issues/375

### CODE REVIEW

**Comment by GeorgianaElena** [comment]
Hi @efedorov-dart! Thanks for spotting this and opening the PR :)
``get_current_user`` became async since https://github.com/jupyterhub/jupyterhub/pull/2142 and indeed FirstUseAuthenticator hasn't been updated accordingly.

While awaiting on this new coroutine is perfectly fine too, I believe it will be better to get the current user directly, using the ``current_user`` accessor (also added in https://github.com/jupyterhub/jupyterhub/pull/2142):
```
user = self.current_user
```
What do you think?

**Comment by efedorov-dart** [comment]
I thank that this project lacks of many unit tests. 
Which causes errors like this go to release.

File: firstuseauthenticator/firstuseauthenticator.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def _register_template_path(self):
 
     @web.authenticated
     async def post(self):
-        user = self.get_current_user()
+        user = await self.get_current_user()
```
Comment (GeorgianaElena) [inline firstuseauthenticator/firstuseauthenticator.py]:
```suggestion
        user = self.current_user
```

**Comment by GeorgianaElena** [comment]
@efedorov-dart, we're doing our best to keep everything up to date. We always welcome and appreciate PRs that can help us improve the process.

This can be merged once it uses `self.current_user` instead of `get_current_user()`.

**Comment by consideRatio** [comment]
Closing as #23 implemented this fix, now we need to make a release.

**Pull request closed without merging**